packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1398,6 +1413,7 @@ class _TimePickerInputState extends State<_TimePickerInput> with RestorationMixi
       _selectedTime.value = TimeOfDay(hour: newHour, minute: _selectedTime.value.minute);
       widget.onChanged(_selectedTime.value);
     }
+    FocusScope.of(context).requestFocus(_minuteFocusNode);
```

Comment:
Again, I think this could just be: 

```dart
FocusScope.of(context).nextFocus()
```

and should probably be moved into the if statement above as if we don't have a valid hour we should stay put.

Reply:
`FocusScope.of(context).nextFocus().` simply unfocus the hour field hence I am requesting minute focus.